core/src/test/java/io/atomix/core/tree/impl/DocumentTreeTest.java

```diff
@@ -300,7 +300,7 @@ public void testClear() throws Exception {
   /**
    * Tests listeners.
    */
-  @Test
+  @Test(timeout = 45000)
```

Comment:
Can we just create a base class that assigns a timeout to all the tests?

Reply:
Not to first look, only thing I see is per class at suite level.

Reply:
This guy is the main suspect though I was able to reproduce the never ending test like in Travis.